applications/areg-portal/backend/areg_portal/settings.py

```diff
@@ -186,5 +189,8 @@
 CATALOG_NUMBER_MAX_LEN = 512
 
 VENDOR_MAX_LEN = 512
-CSRF_TRUSTED_ORIGINS = ['https://www.areg.local','https://areg.local', 'https://areg.dev.metacell.us','https://areg.stage.metacell.us','https://areg.demo.metacell.us','https://antibodyregistry.org/', 'https://www.antibodyregistry.org/']
+
+# Add some specific origins to the CH generated origins
+CSRF_TRUSTED_ORIGINS += ['https://antibodyregistry.org/', 'https://www.antibodyregistry.org/']
```

Comment:
Why do we need those?

Reply:
lol you added those (https://github.com/MetaCell/scicrunch-antibody-registry/commit/be9ed73dfe4f9dfd789bb8a2cbf42bfe1ebacd8e) let me know if we don't need them then I'll remove them

Reply:
that was trying to be a comprehensive list of all possible addresses in the future, but we won't need it after the ch change